test/decompile/basic.txt

```diff
@@ -79,9 +79,9 @@ data d_a(offset: 0) = "Hello, World!\0a\00";
 import function ns_fi();
 
 export function f(a:int, b:int):int {
-  var c:long = 8L;
-  var d:float = 6f;
-  var e:double = 7d;
+  c:long = 8L;
```

Comment:
wait where'd the vars go

Reply:
oops regression not caught by blind test re-gen..